packages/userstorage/src/browser/user-storage-service.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import { Event, Disposable } from '@theia/core/lib/common';
+
+export const UserStorageService = Symbol('UserStorageService');
+
+export interface UserStorageService extends Disposable {
```

Comment:
I think it is better to use `Uri` as a type instead of `string`, looking at clients and implementors it fits better

Reply:
Agreed, I didn't think of that when I moved it from backend to frontend (I think it's because we were using json-rpc for this before that we used strings).